src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,136 @@
+package com.wolox.training.models;
+
+import jakarta.persistence.*;
+import org.springframework.lang.NonNull;
+
+@Entity
+@Table(name = "books")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @NonNull
```

Comment:
For the validation provided by @NonNull to occur at the database level, we should add the configuration of each column so that it does not accept nulls in the db.

Each field should go with the following annotation added:

@Column(nullable = false)


@NonNull checks that there are no nulls at the Java level, that is, before a request to the db is generated. This is perfect since it would avoid communications with the db with incorrect data

```suggestion
    @NonNull
    @Column(nullable = false)
```